Tasks/KubernetesV1/Tests/TestSetup.ts

```diff
@@ -39,7 +39,7 @@ tr.setInput('versionOrLocation', process.env[shared.TestEnvVars.versionOrLocatio
 tr.setInput('versionSpec', process.env[shared.TestEnvVars.versionSpec] || "1.13.2");
 tr.setInput('checkLatest', process.env[shared.TestEnvVars.checkLatest] || "false");
 tr.setInput('specifyLocation', process.env[shared.TestEnvVars.specifyLocation] || "");
-tr.setInput('outputFormat', process.env[shared.TestEnvVars.outputFormat] || 'json');
+tr.setInput('outputFormat', process.env[shared.TestEnvVars.outputFormat]);
```

Comment:
Why was ` || 'json'` removed? Default value is still json, right?

Reply:
It is removed from test setup and add in before each. This helped me to write test cases. For empty string it doesn't hit the intended code. Instead passes json hence had to shift the assignment to before each.